CHANGELOG.md

```diff
@@ -1,20 +1,22 @@
 # Lotus changelog
 
-# v1.28.0 / 2024-07-23
-This is the MANDATORY Lotus v1.28.0 release, which will deliver the Filecoin network version 23, codenamed Waffle 🧇.
+# v1.28.1 / 2024-07-24
 
-**This release sets the Mainnet to upgrade at epoch 4154640, corresponding to 2024-08-06T12:00:00Z.** 
+This is the MANDATORY Lotus v1.28.1 release, which will deliver the Filecoin network version 23, codenamed Waffle 🧇. v1.28.1 is also the minimal version that supports nv23.
+**This release sets the Mainnet to upgrade at epoch 4154640, corresponding to 2024-08-06T12:00:00Z.**
 
 ## ☢️ Upgrade Warnings ☢️
 - If you are running the `v1.26.x` version of Lotus, please go through the `Upgrade Warnings` section for the `v1.27.*` releases, before upgrading to this RC.
-
+- Note that v1.28.0 needed a bug fix and a feature enhancement to ensure a smooth support for nv23 and it was retracted. Please update your node to v1.28.1 or above before the nv23 upgrade!
 - This upgrade includes an additional migration to the events database. Node operators running Lotus with events turned on (off by default) may experience some delay in initial start-up of Lotus as a minor database migration takes place. See [filecoin-project/lotus#12080](https://github.com/filecoin-project/lotus/pull/12080) for full details.
+- All SPs MUST finish onboarding all sectors that have deal IDs in the PreCommitSectors OnChainSectorInfos before upgrading the lotus miner OR THEY WILL BE WASTED. Please see more details in the next section.
 
 ## The Filecoin network version 23 delivers the following FIPs:
 
 - [FIP-0065: Ignore built-in market locked balance in circulating supply calculation](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0065.md)
 - [FIP-0079: Add BLS Aggregate Signatures to FVM](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0079.md)
 - [FIP-0084: Remove Storage Miner Actor Method ProveCommitSectors](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0084.md)
+  -  :warning: Please note that ProveCommitSectors is deprecated in favor of ProveCommitSectors3, which was introduced in FIP-0076. ProveCommitSectors3 will reject the activation of sectors that were precommitted with deal IDs. Storage providers should ensure that their pipeline is updated to adopt the ProveCommitSector3 flow as soon as possible. Otherwise, they risk losing deal collateral, PCD, and sealing work for sectors that were not fully committed on-chain before the upgrade yet have deal IDs in the precommitted sector on-chain info. This release removes the deprecated ProveCommit pipeline, and the new pipeline is fully supported. If you have any questions, please don't hesitate to reach out in #fil-curio-dev.
```

Comment:
```suggestion
  -  :warning: Please note that onboarding via `ProveCommitSectors` is deprecated in favor of `ProveCommitSectors3`, which was introduced in FIP-0076 and activated in the last network upgrade (NV22). `ProveCommitSectors3` will reject the activation of sectors that were precommitted with deal IDs. Storage Providers should ensure that their pipeline is updated to adopt the `ProveCommitSector3` flow as soon as possible. Otherwise, they risk losing deal collateral, PCD, and sealing work for sectors that were not fully committed on-chain before the upgrade yet have deal IDs in the precommitted sector on-chain info. This release removes the deprecated `ProveCommitSectors` pipeline, and the new pipeline is fully supported. Prior to this release, the onboarding pipeline would still prefer to use `ProveCommitSectors` for sectors containing deals. If you have any questions, please don't hesitate to reach out in #fil-curio-dev.
```